_components/CodeSamples.tsx

```diff
@@ -0,0 +1,266 @@
+// {% extends "base.html.jinja" %}
+import React, { useState } from "react";
+import preval from "babel-plugin-preval/macro";
+import { usePathPrefix } from "@redocly/developer-portal/ui";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import Dropdown from "react-bootstrap/Dropdown";
+// {% block bodyclassNamees %}no-sidebar{% endblock %}
+// {% block mainclassNamees %}landing page-community{% endblock %}
+
+// {% block main %}
+
+interface CodeSample {
+  href: string;
+  title: string;
+  description: string;
+  langs: Array<string>;
+}
+// TODO: need to be updated once repo is migrated
+const githubLink = "https://github.com/amarantha-k/xrpl-org-dev-portal";
```

Comment:
@mDuo13 do you know of a better place to store a variable like this in the file structure?